internal/report/store.go

```diff
@@ -14,6 +14,15 @@ import (
 	wgpolicy "sigs.k8s.io/wg-policy-prototypes/policy-report/pkg/api/wgpolicyk8s.io/v1alpha2"
 )
 
+type ReportStore interface {
+	BeforeScanning(ctx context.Context) error
+	AfterScanning(ctx context.Context) error
+	CreateOrPatchPolicyReport(ctx context.Context, policyReport *wgpolicy.PolicyReport) error
+	DeleteOldPolicyReports(ctx context.Context, scanRunID, namespace string) error
+	CreateOrPatchClusterPolicyReport(ctx context.Context, clusterPolicyReport *wgpolicy.ClusterPolicyReport) error
+	DeleteOldClusterPolicyReports(ctx context.Context, scanRunID string) error
+}
+
```

Comment:
Refactor this for something more meaningful. As the SUSE Obs is not actually storing the reports...